sqldelight-core-integration-tests/src/test/kotlin/com/example/TeamQueries.kt

```diff
@@ -69,15 +68,11 @@ class TeamQueries(private val queryWrapper: QueryWrapper, private val database:
         private val coach: String,
         statement: SqlPreparedStatement,
         mapper: (SqlResultSet) -> T
-    ) : Query<T>(statement, teamForCoach, mapper) {
-        fun dirtied(coach: String): Boolean = true
-    }
+    ) : Query<T>(statement, teamForCoach, mapper)
 
     private inner class ForInnerType<out T : Any>(
         private val inner_type: Shoots.Type?,
```

Comment:
these should also be removed now.

Reply:
I need them in a follow up where I make queries entirely lazy

Reply:
hmmm. Why do we want that? Are queries also going to be bound to similar to statements?

Reply:
Yes. It just moves statement creation to first use rather than when created.

See #911 for why

Reply:
Are you going to be making it so the call to `db.getConnection()` happens during the `execute` phase of a query? I'm assuming the answer is yet. Regardless lgtm